src/base/datetime/calendar/body/index.ts

```diff
@@ -285,33 +286,23 @@ export class BaseDateTimeCalendarBody extends KucBase {
                 const dateParts = weekDate.text.split("-");
                 const isSameDate = this._isSameDayOfMoment(dateParts);
                 const isThisMonth = parseInt(dateParts[1], 10) === this._month;
+                const isFocus =
+                  (this.value === weekDate.attr || isSameDate) && isThisMonth;
                 return html`
                   <td
                     role="gridcell"
-                    class="kuc-base-datetime-calendar-body__table__date${(this
-                      .value === weekDate.attr ||
-                      isSameDate) &&
-                    isThisMonth
+                    class="kuc-base-datetime-calendar-body__table__date${isFocus
                       ? "--selected"
-                      : ""}"
+                      : ""}${this._getDateClass(dateParts, isThisMonth)}"
+                    aria-selected="${this.value === weekDate.attr}"
+                    tabindex="${isFocus ? "0" : "-1"}"
+                    aria-current="${this._isToday(dateParts) ? "date" : false}"
+                    aria-label="${dateParts[2]} ${monthString}"
+                    data-date="${weekDate.attr}"
+                    @click="${this._handleClickDateBtn}"
+                    @keydown="${this._handleKeyDownDateBtn}"
```

Comment:
please change function name "_handleClickDateBtn" and  "_handleKeyDownDateBtn", now It's not a button